Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node as build
+
+WORKDIR /code
+COPY . /code
+RUN mv /code/config.example.js /code/config.js
```

Comment:
this should be still a manual step I think

Reply:
sure. there is lot more to do.
but it's still a quick example from cccamp.